core/templates/pages/admin-page/roles-tab/admin-roles-tab.directive.ts

```diff
@@ -381,7 +385,9 @@ angular.module('oppia').directive('adminRolesTab', [
 
         ctrl.clearReviewersData = function() {
           ctrl.contributionReviewersDataFetched = false;
-          ctrl.result = {};
+          ctrl.resultRolesVisible = false;
+          ctrl.userRolesResult = {};
```

Comment:
why `clearReviewersData` is clearing `userRolesResult`?

Reply:
Probably, "clearViewData" would be more descriptive since it is clearing data from the "View User Roles" and "View Contributor Dashboard Users" sections.

Reply:
Maybe have a separate function for each? Or change this to `clearResults`? (I would prefer the first one because it helps us isolate the two forms/functions.)